cli.js

```diff
@@ -73,16 +73,20 @@ function filterProcesses(input, processes, flags) {
 		});
 }
 
-function handleFkillError(proc) {
-	inquirer.prompt([{
-		type: 'confirm',
-		name: 'forceKill',
-		message: 'Error killing process. Would you like to use the force?'
-	}]).then(answer => {
-		if (answer.forceKill === true) {
-			return fkill(proc, {force: true});
-		}
-	});
+function handleFkillError(processes) {
+	if (process.stdout.isTTY === false) {
+		console.log(`Error killing process. Try \`fkill --force ${processes}\``);
```

Comment:
Super nitpick, but use `process` word if there are only one process, otherwise `processes`, in `Error killing process`.